pkg/icingadb/db.go

```diff
@@ -506,7 +506,9 @@ func (db *DB) BatchSizeByPlaceholders(n int) int {
 // YieldAll executes the query with the supplied scope,
 // scans each resulting row into an entity returned by the factory function,
 // and streams them into a returned channel.
-func (db *DB) YieldAll(ctx context.Context, factoryFunc contracts.EntityFactoryFunc, query string, scope interface{}) (<-chan contracts.Entity, <-chan error) {
+func (db *DB) YieldAll(
+	ctx context.Context, factoryFunc contracts.EntityFactoryFunc, query string, namedQueryParams bool, scope ...interface{},
+) (<-chan contracts.Entity, <-chan error) {
```

Comment:
... and avoid the extra parameter?

Reply:
NO! Since it's fine to pass slice of named structs as well.

Reply:
As scope?

Reply:
Yes!

Reply:
Where do we do this?

Reply:
Didn't say we were doing that somewhere right now. Just saying that it's possible!

Reply:
Do we do this in Icinga DB?

Reply:
👇
> Didn't say we were doing that somewhere right now. Just saying that it's possible!

Reply:
Do we do this in Icinga DB? Yes or no? And where?